### PR TITLE
fix(menu-toggle): added support for validation/status

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly/issues/6243

### CODE REVIEW

File: src/patternfly/components/MenuToggle/menu-toggle.scss [hunk 1]
```diff
@@ -163,6 +163,21 @@
   // * Menu toggle small
   --#{$menu-toggle}--m-small--PaddingTop: var(--pf-t--global--spacer--xs);
   --#{$menu-toggle}--m-small--PaddingBottom: var(--pf-t--global--spacer--xs);
+
+  // Status icon
+  --#{$menu-toggle}__status-icon--MarginInlineEnd: var(--pf-t--global--spacer--md);
+
+  // Success
+  --#{$menu-toggle}--m-success--after--BorderBottomColor: var(--pf-t--global--border--color--status--success--default);
```
Comment (thatblindgeye) [inline src/patternfly/components/MenuToggle/menu-toggle.scss:171]: Same would apply to warning and danger vars + where they're used:


```suggestion
  --#{$menu-toggle}--m-success--BorderColor: var(--pf-t--global--border--color--status--success--default);
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: src/patternfly/components/MenuToggle/menu-toggle.scss (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/MenuToggle/menu-toggle.scss]: d'oh! copy/pasta fail 🤦‍♂️

**Review by thatblindgeye — APPROVED** [review]

**Review by andrew-ronaldson — APPROVED** [review]
Tis a fine PR it is!

**Review by lboehling — CHANGES_REQUESTED** [review]
I am noticing that there isn't as much padding to the right of the caret compared to the other menu toggles, but otherwise looks good!

**Review by srambach — APPROVED** [review]
Looks good. Definitely a candidate for a focus ring adjustment but that'll come later. 🚀

**Comment by srambach** [comment]
> I am noticing that there isn't as much padding to the right of the caret compared to the other menu toggles, but otherwise looks good!

I was trying to compare the caret spacing too, but decided design could weigh in on that!  Can always tweak that later if it bugs anyone.

**Comment by mcoker** [comment]
Opened https://github.com/patternfly/patternfly/issues/6458 to track that spacing update from https://github.com/patternfly/patternfly/pull/6441#pullrequestreview-1955693920

**Pull request merged**